src/main/java/frc/robot/RobotContainer.java

```diff
@@ -108,6 +108,16 @@ public RobotContainer() {
    * edu.wpi.first.wpilibj2.command.button.JoystickButton}.
    */
   private void configureButtonBindings() {
+    configureDriveCommands();
+    // configureTestCommands();
+  }
+
+  private void configureTestCommands() {
+    Command fieldRelative = new FieldRelativeDrive(m_swerveDrive, m_driver);
+    m_swerveDrive.setDefaultCommand(fieldRelative);
+  }
+
+  private void configureDriveCommands() {
```

Comment:
Maybe call this one something else? Since it's not just drive controls, but also operator controls.
Blanking on a better name though

Reply:
I wasn't that sure either... something like "match" commands?

Reply:
That works or maybe Main commands?